A created project comes with linting issues

I chose to create a React project with no testing and with the airbnb configuration (created with `npx @neutrinojs/create-project ci-metrics`.
After that I would run `yarn start` and I get linting errors.

```
armenzg@armenzg-mbp ci-metrics$ yarn start
yarn run v1.3.2
$ neutrino start
✔ Development server running on: http://localhost:5000
✔ Build completed

ERROR in ./src/index.jsx
error: Expected a newline after '(' (function-paren-newline) at src/index.jsx:6:26:
  4 |
  5 | const root = document.getElementById('root');
> 6 | const load = () => render((
    |                          ^
  7 |   <AppContainer>
  8 |     <App />
  9 |   </AppContainer>


error: Expected a newline before ')' (function-paren-newline) at src/index.jsx:10:8:
   8 |     <App />
   9 |   </AppContainer>
> 10 | ), root);
     |        ^
  11 |
  12 | // This is needed for Hot Module Replacement
  13 | if (module.hot) {


2 errors found.
2 errors potentially fixable with the `--fix` option.
 @ multi (webpack)-dev-server/client?http://localhost:5000 (webpack)/hot/dev-server.js ./node_modules/react-hot-loader/patch.js ./src/index

ERROR in ./src/App.jsx
error: Missing trailing comma (comma-dangle) at src/App.jsx:6:23:
  4 | export default class App extends Component {
  5 |   state = {
> 6 |     name: 'ci-metrics'
    |                       ^
  7 |   };
  8 |
  9 |   render() {


1 error found.
1 error potentially fixable with the `--fix` option.
 @ ./src/index.jsx 4:0-24 15:2-34 15:29-33
 @ multi (webpack)-dev-server/client?http://localhost:5000 (webpack)/hot/dev-server.js ./node_modules/react-hot-loader/patch.js ./src/index
```

I've probably installed dependencies on my early days by mistake [1].
I've brought it down to [2].
I still get linting issues.
I've also removed my linting file [3]
I've tried from scratch with the same results [4]
Here are the versions of the binaries involved:

> armenzg@armenzg-mbp ci-metrics$ node --version
> v8.8.0
> armenzg@armenzg-mbp ci-metrics$ npm --version
> 5.5.1
> armenzg@armenzg-mbp ci-metrics$ npx --version
> 9.6.0
> armenzg@armenzg-mbp ci-metrics$ yarn --version
> 1.3.2




[1]

> armenzg@armenzg-mbp ci-metrics$ npm ls -g --depth=0
> /usr/local/lib
> ├── create-react-app@1.4.3
> ├── eslint@4.11.0
> ├── eslint-config-airbnb@16.1.0
> ├── eslint-plugin-import@2.8.0
> ├── eslint-plugin-jsx-a11y@6.0.2
> ├── eslint-plugin-react@7.4.0
> ├── exp@45.1.0
> ├── js-beautify@1.6.14
> ├── minimatch@3.0.2
> ├── mocha@4.0.1
> ├── npm@5.5.1
> ├── npm-check-updates@2.13.0
> ├── opencollective@1.0.3
> ├── platform-health-dashboard@0.0.1 -> /Users/armenzg/repos/platform-health
> ├── telemetry-next-node@2.0.0 -> /Users/armenzg/repos/telemetry-next-node
> ├── webpack@3.8.1
> └── yarn@1.0.1
> 
> npm ERR! peer dep missing: react@^0.14.0 || ^15.0.0-0 || ^16.0.0-0, required by react-redux@5.0.6
> npm ERR! peer dep missing: jquery@>=1.11.1, required by metrics-graphics@2.11.0
> npm ERR! peer dep missing: superagent@~0.17.0, required by superagent-retry@0.3.0

[2]

> armenzg@armenzg-mbp ci-metrics$ npm ls -g --depth=0
> /usr/local/lib
> ├── npm@5.5.1
> ├── npm-check-updates@2.13.0
> └── yarn@1.0.

[3]
> armenzg@armenzg-mbp ci-metrics$ cat ~/.eslintrc.js
> module.exports = {
>     "extends": "airbnb",
>     "rules": {}
> };

[4]

> armenzg@armenzg-mbp repos$ rm -rf ci-metrics/
> armenzg@armenzg-mbp repos$ npx @neutrinojs/create-project ci-metrics
> npx: installed 210 in 8.007s
>                           _          _
>       _ __    ___  _   _ | |_  _ __ (_) _ __    ___
>      | '_ \  / _ \| | | || __|| '__|| || '_ \  / _ \
>      | | | ||  __/| |_| || |_ | |   | || | | || (_) |
>      |_| |_| \___| \__,_| \__||_|   |_||_| |_| \___/
> 
> Welcome to Neutrino! 👋
> To help you create your new project, I am going to ask you a few questions.
> 
> ? 🤔  First up, what would you like to create? A web or Node.js application
> ? 🤔  Next, what kind of application would you like to create? React
> ? 🤔  Would you like to add a test runner to your project? None
> ? 🤔  Would you like to add linting to your project? Airbnb style rules
> 
> 👌  Looks like I have all the info I need. Give me a moment while I create your project!
> 
>    create ci-metrics/package.json
>    create ci-metrics/.neutrinorc.js
>    create ci-metrics/src/App.css
>    create ci-metrics/src/App.jsx
>    create ci-metrics/src/index.jsx
>    create ci-metrics/.eslintrc.js
> 
> ⏳  Installing dependencies: prop-types, react, react-dom, react-hot-loader
> ⏳  Installing devDependencies: @neutrinojs/react, neutrino, @neutrinojs/airbnb
> ⏳  Performing one-time lint
> 
> Hooray, I successfully created your project!
> 
> I have added a few yarn scripts to help you get started:
>   • To build your project run:  yarn build
>   • To start your project locally run:  yarn start
>   • To lint your project manually run:  yarn lint
>     You can also fix some linting problems with:  yarn lint --fix
> 
> Now change your directory to the following to get started:
>   ci-metrics
> 
> ❤️  Neutrino
> armenzg@armenzg-mbp repos$ cd ci-metrics/

--- a/packages/create-project/commands/init/index.js
+++ b/packages/create-project/commands/init/index.js
@@ -185,10 +185,15 @@ module.exports = class Project extends Generator {
 
     if (this.data.linter) {
       this.log(`${chalk.green('⏳  Performing one-time lint')}`);
-      this.spawnCommandSync(packages.NEUTRINO, ['lint', '--fix'], {
-        stdio: this.options.stdio === 'inherit' ? 'ignore' : this.options.stdio,
-        cwd: this.options.directory
-      });
+      this.spawnCommandSync(packageManager,
+        isYarn
+          ? ['lint', '--fix']
+          : ['run', 'lint', '--fix'],
+        {
+          stdio: this.options.stdio === 'inherit' ? 'ignore' : this.options.stdio,
+          env: process.env,
+          cwd: this.options.directory
+        });
     }
   }
 

